Documentation/tutorials/map-track-data-using-data-prep.md

```diff
@@ -0,0 +1,164 @@
+        "contextdata" : {
+          "product.view.event" : "1",
+          "product.name" : "wide_brim_sunhat",
+          "product.id" : "12345"
+        }
+      }
+    }
+  ]
+}
+```
+
+</p></details>
+
+To capture properties from Experience events, extract the JSON object under the **events** array object, as Data Prep requires that the **data** and **xdm** objects are top-level objects in the provided JSON source. If the JSON is valid, a preview schema is displayed in the right-hand panel. Select **Next** to continue.
+
+![select data](../assets/select-data-trackstate.png)
+
+## Mapping
+From the **Mapping** step, fields from the source data are mapped to the target fields of the XDM schema in Experience Platform.
+
+Select **Add new mapping** to create a new mapping row.
+
+![add new mapping](../assets/add-new-mapping.png)
+
+Select the source icon (![source icon](../assets/source-icon.png)), and select the source field from the dialog that appears. Use the **Select** button to continue.
+
+![source mapping](../assets/source-mapping.png)
+
+Now select the schema icon (![schema icon](../assets/schema-icon.png)), and select the target schema field from the dialog that appears. Use the **Select** button to continue.
+
+![target mapping](../assets/target-mapping.png)
+
+The mapping page now displays the source to target field mapping. The **Mapping progress** section displays the progress of the total number of source fields mapped.
+> **Warning**
+> Source fields which contain periods need the periods escaped, otherwise the mapping service will interpret the field as nested objects. The source field may be manually edited to escape the periods, as in `data.contextdata.product\.name`.
+
+![field mapped](../assets/field-mapped.png)
+
+Continue the above steps to map additional fields to the target schema. It is not required to map all the source fields to the target schema, however any required source fields must be mapped. The **Required fields** counter displays the number of required source fields which are yet to be mapped. Once all the required source fields are mapped, select **Save** to complete the mapping.
+
+![mapping complete](../assets/mapping-complete.png)
+
+The mapping process may be repeated for additional source data if needed by selecting the **Add JSON** button.
+
+![select data](../assets/select-data-trackaction.png)
+
+> **Note**
+> XDM source fields are automatically mapped if the same field appears in the target schema. For example, the fields _xdm.\_id_ and _xdm.timestamp_ are required fields in a time-series XDM schema so you will notice they are automatically mapped from the source data to the target schema and do not require a mapping entry.
+
+> **Note**
+> The Edge Bridge extension automatically sets an _xdm.eventType_ value of _analytics.track_. However, the value may be changed by adding a new mapping row in Data Prep by setting the **Target Field** to "eventType".
+> ![map eventType](../assets/map-eventtype.png)
```

Comment:
there seem to be a broken link here

Reply:
This one still shows as broken, might need to take it outside of the note block to display correctly.